Add maker command to generate data providers/persisters (#3850)
api-platform/core: src/Bridge/Symfony/Maker/Resources/help/MakeDataProvider.txt

> The <info>%command.name%</info> command generates a new data provider class.

Reviewer: ```suggestion
The <info>%command.name%</info> command generates an API Platform data provider class.
```
